stun_server/dns.tf

```diff
@@ -0,0 +1,24 @@
+resource "cloudflare_load_balancer" "stun-server" {
```

Comment:
As this PR is "migrate," I would expect to see import blocks. Is this not a migration?
But a replication?